walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -8,21 +8,21 @@
   <body>
     <div id="content">
       <h1>Jonathan Aner's Portfolio</h1>
-      <p>Welcome to my Portfolio</p>
+      <h2>Welcome to my Portfolio</h2>
```

Comment:
To confirm: if you want to keep the headings the same size, you can use the h2 (or whichever h# tag size looks the best) tag throughout. Otherwise, please update the screenshot with the new headings sizes!